README.md

```diff
@@ -261,33 +255,23 @@ See the `mac` script for examples.
 Laptop functions such as `fancy_echo` and `gem_install_or_update` can be used
 in your `~/.laptop.local`.
 
-How to manage background services (Redis, Postgres, MySQL)
+What about background services (Redis, Postgres, MySQL, etc.)?
 ----------------------------------------------------------
-The script does not automatically launch these services after installation
-because you might not need or want them to be running. With Homebrew Services,
-starting, stopping, or restarting these services is as easy as:
-
-```
-brew services start|stop|restart [name of service]
-```
-
-For example:
-
-```
-brew services start redis
-```
+The script does not automatically install services like databases because you
+may not need any particular one, or you may need specific versions, not just
+the latest.  For services, we recommend using Docker.  For example, you can
```

Comment:
Not necessary, but a link to Docker's docs might be nice on the off chance someone is unfamiliar with it.